modules/microshift-accessing.adoc

```diff
@@ -3,7 +3,9 @@
 // microshift/microshift-install-rpm.adoc
 
 :_content-type: CONCEPT
-[id="accssing-microshift-cluster_{context}"]
+[id="accessing-microshift-cluster_{context}"]
 = Accessing the {product-title} cluster
 
-Use the procedures in this section to access the {product-title} cluster locally, either from the same machine running the {product-title} service or remotely from a different machine running the {product-title} service.
\ No newline at end of file
+Use the procedures in this section to access the {product-title} cluster locally, either from the same machine running the {product-title} service or remotely from a different machine running the {product-title} service.
```

Comment:
I understand the end of this sentence as "from a different machine [that] is running the MicroShift service" (= the MicroShift host), whereas I think we mean to say "from a machine different from the one running the MicroShift machine" (= the workstation).